t-circulating-supply/main.py

```diff
@@ -92,7 +92,7 @@ def main(request):
 
     # NU Tokens Calc
     nu_circulating_supply = NU_CIRCULATING_SUPPLY_PRE_NUCO_VESTING_DATE
-    if maya.now() >= NUCO_VESTING_DATE:
+    if datetime.datetime.now() >= NUCO_VESTING_DATE:
```

Comment:
```suggestion
    if datetime.datetime.now(datetime.timezone.utc) >= NUCO_VESTING_DATE:
```

Reply:
this part i don't think matters - now is the same time in any timezone; it gets boiled down to milliseconds since the unix epoch. I'll put in the other one though!

Reply:
`datetime` has issues with non-offset aware datetimes.

eg.

```python
In [128]: d = datetime.datetime(2025, 10, 15, 0, 0, 0, tzinfo=datetime.timezone.utc)

In [129]: now = datetime.datetime.now()

In [130]: now >= d
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[130], line 1
----> 1 now >= d

TypeError: can't compare offset-naive and offset-aware datetimes
```

Reply:
oh woah :o

Reply:
62b7432ab2e8a29d845c211af7a32b3a85078c79